src/main/scala/com/github/traviscrawford/spark/dynamodb/ParsedFilterExpression.scala

```diff
@@ -0,0 +1,44 @@
+package com.github.traviscrawford.spark.dynamodb
+
+/** Simple parser for filter_expression options.
+  *
+  * This will add Dynamo indirection so that reserved keywords can
+  * be handled in filter expressions.
+  *
+  * Regex matching inspired by:
+  * https://ikaisays.com/2009/04/04/using-pattern-matching-with-regular-expressions-in-scala/
+  */
+private object ParsedFilterExpression {
```

Comment:
Provide limited parsing support to get around field names that are used as Dynamo keywords. Emphasis on __*limited*__. My goal wasn't to reverse engineer the Dynamo expression parser, just cover the 80/20 cases. If the parser can't handle the filter expression, it just passes the whole thing, verbatim, to Dynamo.